tox.ini

```diff
@@ -94,7 +93,8 @@ skipsdist = True
 skip_install = True
 deps =
     bandit==1.7.0
-commands = bandit -r aea benchmark examples packages \
+commands = bandit -r aea benchmark examples packages --exclude packages/*/*/*/tests \
```

Comment:
what's the problem there?

Reply:
`assert` statements must be ignored; so we separate bandit check on source code and bandit check on test code

Reply:
I guess ideally we just ignore that single check for asserts, not the files